main/fossil/APKBUILD

```diff
@@ -1,16 +1,17 @@
 # Contributor: Paul Onyschuk <blink@bojary.koba.pl>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=fossil
-pkgver=2.3
-pkgrel=1
+pkgver=2.4
+pkgrel=0
 pkgdesc="Simple, high-reliability, distributed software configuration management"
 url="http://www.fossil-scm.org"
 arch="all"
 license="BSD"
 depends=""
-makedepends="libressl-dev zlib-dev sqlite-dev"
+makedepends="libressl-dev sqlite-dev tcl-dev zlib-dev"
 source="http://www.fossil-scm.org/index.html/uv/$pkgname-src-$pkgver.tar.gz"
 builddir="$srcdir"/$pkgname-$pkgver
+options="!check"
```

Comment:
Please add a comment explaining why you disabled check.

Reply:
Done